lib/ansible/modules/deb822_repository.py

```diff
@@ -357,6 +370,21 @@ def write_signed_by_key(module, v, slug):
     return changed, filename, None
 
 
+def install_python_debian(module, deb_pkg_name):
+
+    if not module.check_mode:
+        apt_get_path = module.get_bin_path('apt-get')
```

Comment:
at this point we can just use `apt`, apt-get is currently a backwards compatibility shim and might go away

Reply:
no, that module is for 'older' debian versions, it was created before `apt` was a thing, this helps distinguish the versions these modules apply to.